### PR TITLE
:bug: Fixed regexp for filtering carousel publishes

### PR DESCRIPTION
**Pull request by izzyblues**

### CODE REVIEW

File: app_test.go [hunk 1]
```diff
@@ -5,8 +5,8 @@ import (
 )
 
 const syntheticTID = "SYNTHETIC-REQ-MONe4d2885f-1140-400b-9407-921e1c7378cd"
-const carouselRepublishTID = "tid_ofcysuifp0_carousel_1488384556"
-const carouselGeneratedTID = "tid_ofcysuifp0_carousel_1488384556_gentx"
+const carouselRepublishTID = "something123_carousel_1488384556"
+const carouselGeneratedTID = "something-else-321_carousel_1488384556_gentx"
 const naturalTID = "tid_xltcnbckvq"
```
Comment (sjwells) [inline app_test.go]: Wouldn't a generatedTID still start with tid_{uuid}?

Comment (galiri) [inline app_test.go]: I think you could have added a test to cover the non tid case,  that is covered here, just would have been clearer.

Comment (izzyblues) [inline app_test.go]: Good point... I am going to put realistic transaction IDs.

File: transaction_id_check_test.go [hunk 2]
```diff
@@ -5,8 +5,9 @@ import (
 )
 
 const syntheticTID = "SYNTHETIC-REQ-MONe4d2885f-1140-400b-9407-921e1c7378cd"
-const carouselRepublishTID = "something123_carousel_1488384556"
-const carouselGeneratedTID = "something-else-321_carousel_1488384556_gentx"
+const carouselRepublishTID = "tid_ofcysuifp0_carousel_1488384556"
+const carouselUncoventionalRepublishTID = "republish_-10bd337c-66d4-48d9-ab8a-e8441fa2ec98_carousel_1493606135"
```
Comment (khatton-ft) [inline transaction_id_check_test.go]: :pencil2: Should be `Unconventional`

Comment (izzyblues) [inline transaction_id_check_test.go]: 😭

**Pull request merged**